tensorboard/main.py

```diff
@@ -207,16 +203,18 @@ def run_simple_server(tb_app):
     # An error message was already logged
     # TODO(jart): Remove log and throw anti-pattern.
     sys.exit(-1)
-  msg = 'Starting TensorBoard %s at %s' % (version.VERSION, url)
-  print(msg)
-  tf.logging.info(msg)
-  print('(Press CTRL+C to quit)')
-  sys.stdout.flush()
-
-  server.serve_forever()
+  logger = base_logging.getLogger('tensorflow' + util.LogHandler.EPHEMERAL)
+  logger.setLevel(base_logging.INFO)
+  logger.info('TensorBoard %s at %s (Press CTRL+C to quit) ',
+              version.VERSION, url)
+  try:
+    server.serve_forever()
+  finally:
+    logger.info('')
```

Comment:
It also clears the ephemeral state, so the last ephemeral line stops being sticky.